packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -645,12 +645,29 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
           isRead: false
         };
         draft.chatThreads.unshift(newChatThread);
+        // lazy load more info
+        void this.loadChatDetails(event.message.chatId);
```

Comment:
I'm looking for the lazy load pattern when I see this comment.  Can this just be "load more info"?

Reply:
Your right, its not so much lazy as async.